src/engine/strat_engine/backstore/blockdev.rs

```diff
@@ -59,6 +61,7 @@ impl StratBlockDev {
         upper_segments: &[(Sectors, Sectors)],
         user_info: Option<String>,
         hardware_info: Option<String>,
+        key_description: Option<String>,
```

Comment:
Please add some docs for the new parameter.

Reply:
Resolved.